documents/agent-setup.md

```diff
@@ -0,0 +1,137 @@
+|                       | `didOpts`           | Options related to the Decentralized Identifier (DID) of the issuer.                                                                                                                                                      |
+|                       | `userPinRequired`   | (optional) Boolean indicating if a user PIN is required for operations.                                                                                                                                                   |
+|                       | `cNonceExpiresIn`   | (optional) Number specifying the expiration time for the client nonce in the issuance process.                                                                                                                            |
+
+#### 4. oid4vp_options <a name="oid4vp_options"></a>
+
+Here, you will store configuration files related to OpenID for Verifiable Presentations (OID4VP).
+
+| Variable           | Sub-Variable          | Description                                                                                             |
+|--------------------|-----------------------|---------------------------------------------------------------------------------------------------------|
+| `definitionId`     |                       | Unique identifier associated with a specific credential definition.                                     |
+| `rpOpts`           |                       | Configuration options for the Relying Party (RP).                                                       |
+|                    | `responseMode`        | (optional) Specifies the method that the RP uses to receive the response from the OpenID Provider.      |
+|                    | `supportedVersions`   | (optional) Lists the versions supported by the RP.                                                      |
+|                    | `sessionManager`      | (optional) An instance of IRPSessionManager from Sphereon's SIOP-OID4VP library managing session state. |
+|                    | `expiresIn`           | (optional) Defines the duration after which the RP's request or session should expire.                  |
+|                    | `eventEmitter`        | (optional) An object used to handle events, enabling asynchronous event-driven programming.             |
+|                    | `didOpts`             | Options related to the Decentralized Identifier (DID) of the verifier.                                  |
+
+#### 5. presentation_definitions <a name="presentation_definitions"></a>
+
+The content of this directory is essential for defining how the agent understands and processes presentation requests. For a more thorough description of presentation_definition in [presentation exchange documentation](https://identity.foundation/presentation-exchange/#presentation-definition), and how it's being used in [OID4VP](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html) spec.
```

Comment:
Explain that the presentation definition is relevant in scenario's where the demo ask a user for it's credentials from the wallet

Reply:
added the explanation